src/Forms/Shared/Samples/UtilityNetwork/DisplayUtilityNetworkContainer/readme.metadata.json

```diff
@@ -0,0 +1,40 @@
+{
+    "category": "Utility network",
+    "description": "A utility network container allows a dense collection of features to be represented by a single feature, which can be used to reduce map clutter.",
+    "formal_name": "DisplayUtilityNetworkContainer",
+    "ignore": false,
+    "images": [
+        "DisplayUtilityNetworkContainer.jpg"
+    ],
+    "keywords": [
+        "associations",
+        "connectivity association",
+        "containment association",
+        "structural attachment associations",
+        "utility network"
+    ],
+    "nuget_packages": {
```

Comment:
Depending on when this is likely to be merged relative to the PR to remove the nuget_packages section, maybe consider removing this?